.github/workflows/ci.yaml

```diff
@@ -541,6 +541,8 @@ jobs:
           fetch-depth: 0
           ref: ${{ github.event.pull_request.head.sha }}
 
+      - uses: ./.github/actions/handle-tagged-build
+
```

Comment:
I don't think we need this here. This job only runs on pushes to the `master` branch.